### PR TITLE
Resolves Housing Issues

### PR DESCRIPTION
**Pull request by NiiRoZz**
#769

- Disable storagebox collision with players.
- Check to make sure you're within a range of your house (Prevent running outside with the crate...).
- Method to cancel preview (escape key).
- Removed the house search on house menu, it's been moved to the container level.

### CODE REVIEW

**Comment by setoy** [comment]
> Removed the house search on house menu, it's been moved to the container level.
How do you open the door as a cop to be able to search the container?

File: life_client/stringtable.xml [hunk 1]
```diff
@@ -813,15 +813,15 @@
...
-            <Original>Hit spacebar to place the container.</Original>
-            <Czech>Hit mezerníku na místo Kontejner.</Czech>
-            <French>Appuyez sur espace pour placer le conteneur.</French>
-            <Spanish>Aprieta la barra de espacio para colocar el Contenedor.</Spanish>
-            <Italian>Premi barra spaziatrice per posizionare il contenitore.</Italian>
-            <Polish>Hit spacji, aby umieścić pojemnik.</Polish>
-            <Portuguese>Pressione a barra de espaço para posicionar.</Portuguese>
-            <Russian>Хит пробел, чтобы поместить контейнер.</Russian>
-            <German>Leertaste drücken, um den Container zu platzieren.</German>
+            <Original>Hit spacebar to place the container or ESC for stop preview.</Original>
```
Comment (Jawshy) [inline life_client/stringtable.xml]: ` or ESC to stop preview.` is more grammatically accurate.

**Comment by NiiRoZz** [comment]
@setoy fn_raidHouse it's only linked to search inventory illegal item inside house, this system of raid doesn't needed at all, you can breakdoor with "Break down door" action

File: life_client/stringtable.xml [hunk 2]
```diff
@@ -813,15 +813,15 @@
...
-            <German>Leertaste drücken, um den Container zu platzieren.</German>
+            <Original>Hit spacebar to place the container or ESC to stop preview.</Original>
+            <Czech>Hit mezerníku na místo Kontejner, ESC pro zastavení náhledu.</Czech>
+            <French>Appuyez sur espace pour placer le conteneur ou ESC pour arrêter de placer le conteneur.</French>
+            <Spanish>Aprieta la barra de espacio para colocar el Contenedor, ESC para la parada de previsualización.</Spanish>
+            <Italian>Premi barra spaziatrice per posizionare il contenitore, ESC per arresto anteprima.</Italian>
+            <Polish>Hit spacji, aby umieścić pojemnik, ESC do podglądu zatrzymania.</Polish>
+            <Portuguese>Pressione a barra de espaço para posicionar, ESC para visualização parada.</Portuguese>
+            <Russian>Хит пробел, чтобы поместить контейне, ESC для остановки предварительного просмотра.</Russian>
+            <German>Leertaste drücken, um den Container zu platzieren, ESC für Stop Vorschau.</German>
```
Comment (setoy) [inline life_client/stringtable.xml]: <German>Leertaste drücken, um den Container zu platzieren. ESC zum Beenden der Vorschau.</German>

**Comment by NiiRoZz** [comment]
@setoy you can check grammar german on last commit please ?

**Comment by setoy** [comment]
@NiiRoZz Looks fine! :)

**Comment by BoGuu** [comment]
This doesn't fix collisions

**Comment by BoGuu** [comment]
This doesn't fix collisions

File: life_client/CfgRemoteExec.hpp [hunk 3]
```diff
@@ -170,5 +170,6 @@ class CfgRemoteExec {
         F(addPrimaryWeaponItem,ANYONE)
         F(addWeapon,ANYONE)
         F(setFuel,ANYONE)
+        F(enableSimulationGlobal,ANYONE)
```
Comment (BoGuu) [inline life_client/CfgRemoteExec.hpp]: Should be server only

File: life_client/core/functions/fn_keyHandler.sqf [hunk 4]
```diff
@@ -50,6 +50,14 @@ if (life_container_active) then {
         case 57: {
             [] spawn life_fnc_placestorage;
         };
+        //ESC key
+        case 1: {
+            if (!life_container_active || life_container_activeObj isEqualTo objNull || !((typeOf life_container_activeObj) in ["B_supplyCrate_F","Box_IND_Grenades_F"])) exitWith {};
```
Comment (BoGuu) [inline life_client/core/functions/fn_keyHandler.sqf]: lazy eval

File: life_client/core/items/fn_placestorage.sqf [hunk 5]
```diff
@@ -9,23 +9,21 @@
...
-if (!life_container_active) exitWith {};
-if (life_container_activeObj == objNull) exitWith {};
-if (!((typeOf life_container_activeObj) in ["B_supplyCrate_F","Box_IND_Grenades_F"])) exitWith {};
+if (!life_container_active || life_container_activeObj isEqualTo objNull || !((typeOf life_container_activeObj) in ["B_supplyCrate_F","Box_IND_Grenades_F"])) exitWith {};
 
 _container = life_container_activeObj;
 _isFloating = if (((getPos _container) select 2) < 0.1) then {false} else {true};
 detach _container;
-[_container,true] remoteExecCall ["life_fnc_simDisable",RANY];
+[_object,true] remoteExecCall ["enableSimulationGlobal", RSERV];
```
Comment (tkcjesse) [inline life_client/core/items/fn_placestorage.sqf]: _object eh? Thats not even fucking used in the script. Glad we test things... glad we just throw the shit in there. @NiiRoZz @BoGuu @danielstuart14

**Pull request merged**